.github/workflows/test.yaml

```
@@ -13,15 +13,20 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
+    - name: Set Node.js 16.x
+      uses: actions/setup-node@v3.4.1
+      with:
+        node-version: 16.x
+        cache: npm
+
     - name: Install Dependencies
-      run: npm install
+      run: npm ci
```

```suggestion
      run: npm ci
```
I don't know how different `npm ci` and `npm install` but I often face the issue where npm is not available :/

---

the difference is that `npm ci` is basically similar to `yarn install --frozen-lockfile`

https://docs.npmjs.com/cli/v8/commands/npm-ci

In each action I have added setup nodejs with cache.